components/Comment/CommentActions.tsx

```diff
@@ -36,6 +103,15 @@ const styles = StyleSheet.create({
     columnGap: "5px",
     alignItems: "center",
     cursor: "pointer",
+    fontSize: 15,
+    fontWeight: 500,
+  },
+  actionReply: {
+    marginLeft: "auto",
+  },
+  editAction: {},
+  actionText: {
+    color: colors.secondary.text,
```

Comment:
We should update this later, but I think we need to get rid of our big config objects. Because it's an object, all the colors are imported at this import which is causing the bundle size to grow, when we only use this one color here. 

Also, I think it's better if we make these functions so we can pass opacity into the color, or make a separate getter function like 

```
color({opacity: .5, color: secondaryTextColor});
```

and have the default opacity be 1.

Reply:
I do agree that our colors need to be reworked from the ground-up at some point. Ideally we should have a single config entity. 

As for whether it should be an object or function. Not sure. We will need to run some tests. You may be onto something here but not sure if an object adds non-trivial bloat. It may though.

I know in the case of icons it does because many of these icons are quite big. I did a quick test where icons.js was updated to export an empty object literal `{}` instead of one with icons and that reduce the "shared" bundle size by 120kb from 880kb => 760kb.

Reply:
Yeah, we should do it like FontAwesome where it's just named exports, like 

`export secondaryTextColor = ...`

Our API is also a massive one where we'll probably save over 100kb in bundle size just by converting it to named exports.

Maybe a job for GPT4? lol